platform/src/components/aws/cognito-user-pool-client.ts

```diff
@@ -32,6 +32,7 @@ export class CognitoUserPoolClient extends Component implements Link.Linkable {
 
     const providers = normalizeProviders();
     const client = createClient();
+    const callbackUrls = normalizeCallbackUrls();
```

Comment:
otherwise we get "Cannot access 'callbackUrls' before initialization"

```suggestion
    const callbackUrls = normalizeCallbackUrls();
    const client = createClient();
```